components/psst/browser/content/psst_tab_web_contents_observer.cc

```diff
@@ -176,14 +180,17 @@ void PsstTabWebContentsObserver::OnUserScriptResult(
     return;
   }
 
+  timeout_timer_.Stop();
```

Comment:
Shouldn't this be at the top of this method? If `OnUserScriptResult` then it didn't time out, right? If the policy script is not available we don't want to fire the timeout? Separately (for follow-up) in these early return cases shouldn't we be updating the status?